streaming/streaming.go

```diff
@@ -302,42 +306,73 @@ func LaunchStream(ctx context.Context, driver string, dbList []profiles.LogicalS
 }
 
 func CleanUpStreamingJobs(ctx context.Context, conv *internal.Conv, projectID, region string) error {
+	//create clients
 	c, err := dataflow.NewJobsV1Beta3Client(ctx)
 	if err != nil {
 		return fmt.Errorf("could not create job client: %v", err)
 	}
 	defer c.Close()
 	fmt.Println("Created dataflow job client...")
+	dsClient, err := datastream.NewClient(ctx)
+	fmt.Println("Created datastream client...")
+	if err != nil {
+		return fmt.Errorf("datastream client can not be created: %v", err)
+	}
+	defer dsClient.Close()
+
+	//clean up for single instance migrations
+	if conv.Audit.StreamingStats.DataflowJobId != "" {
+		CleanupDataflowJob(ctx, c, conv.Audit.StreamingStats.DataflowJobId, projectID, region)
+	}
+	if conv.Audit.StreamingStats.DataStreamName != "" {
+		CleanupDatastream(ctx, dsClient, conv.Audit.StreamingStats.DataStreamName, projectID, region)
+	}
+	// clean up jobs for sharded migrations (with error handling)
+	for _, dfId := range conv.Audit.StreamingStats.ShardToDataflowJobMap {
+		err := CleanupDataflowJob(ctx, c, dfId, projectID, region)
+		if err != nil {
+			fmt.Printf("Cleanup of the dataflow job: %s was unsuccessful, please clean up the job manually", dfId)
```

Comment:
as a follow up, we should float these errors back to the UI, not just print it out here.

Reply:
Added this an example to the open backlog - https://b.corp.google.com/issues/269275585
We'll need to refactor the code a bit to create DTO boundaries between - 

1. HB - The library
2. HB - The CLI interface
3. HB - The backend service

This will enable taking to the library code in terms of request/response objects and floating back errors to the UI and to the CLI, generically.